draft-ietf-anima-brski-prm.md

```diff
@@ -1103,589 +1166,583 @@ pledgeenrollrequesttrigger = {
-
-{{MASA-REG-vr}} below provides an example of the voucher with two signatures.
+For BRSKi-PRM, the domain registrar MUST add an additional JWS Protected Header and JWS Signature to the MASA-provided Voucher.
+{{rcv_representation}} summarizes the serialization of the JWS-signed JSON Voucher' artifact:
 
 ~~~~
-# The MASA issued voucher with additional registrar signature in
-  General JWS Serialization syntax
 {
-  "payload": BASE64URL(ietf-voucher:voucher),
+  "payload": BASE64URL(JSON Voucher Data),
   "signatures": [
     {
       "protected": BASE64URL(UTF8(JWS Protected Header (MASA))),
-      "signature": BASE64URL(JWS Signature)
+      "signature": BASE64URL(JWS Signature (MASA))
     },
     {
-      "protected": BASE64URL(UTF8(JWS Protected Header (Reg))),
-      "signature": BASE64URL(JWS Signature)
+      "protected": BASE64URL(UTF8(JWS Protected Header (Registrar))),
+      "signature": BASE64URL(JWS Signature (Registrar))
     }
   ]
 }
+~~~~
+{: #rcv_representation title='Voucher\' Representation in General JWS JSON Serialization Syntax' artwork-align="left"}
 
-# Example: Decoded payload "ietf-voucher:voucher" representation in
-  JSON syntax
-{
-  "ietf-voucher:voucher": {
-     "assertion": "agent-proximity",
-     "serial-number": "callee4711",
-     "nonce": "base64encodedvalue==",
-     "created-on": "2022-01-04T00:00:02.000Z",
-     "pinned-domain-cert": "base64encodedvalue=="
-  }
-}
+In BRSKI {{!RFC8995}}, the registrar proves possession of its credential through the server authentication within the TLS session with the pledge.
+While the pledge cannot verify the registrar certificate at the time of TLS session establishment, it can verify the TLS server certificate through the certificate in the `pinned-domain-cert` field upon receiving the Voucher artifact (see {{Section 5.6.2 of !RFC8995}}).
 
-# Example: Decoded "JWS Protected Header (MASA)" representation
-  in JSON syntax
-{
-  "alg": "ES256",
-  "typ": "voucher-jws+json",
-  "x5c": [
-    "base64encodedvalue==",
-    "base64encodedvalue=="
-  ]
-}
+In BRSKI-PRM with the Registrar-Agent mediating all communication, this second signature provides verification and POP of the private key for the registrar LDevID certificate provided in the initial tPVR artifact from the Registrar-Agent (see {{tpvr-artifact}}).
 
-# Example: Decoded "JWS Protected Header (Reg)" representation
-  in JSON syntax
-{
-  "alg": "ES256",
-  "x5c": [
-    "base64encodedvalue==",
-    "base64encodedvalue=="
-  ]
-}
-~~~~
-{: #MASA-REG-vr title='Representation of MASA issued voucher with additional registrar signature' artwork-align="left"}
+Depending on the security policy of the operator, this signature can also be interpreted as explicit authorization of the registrar to install the contained trust anchor. TODO(what exactly is the trust anchor? pinned-domain-cert?)
```

Comment:
TODO

Reply:
Yes, in the context of the voucher it is the pinned domain certificate.